.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
_Note: Since this PR does not include all the files, I'll include my comments here:_

## On your readme file:

- [ ] The "Usage" section includes no content.  Kindly either add some content to it or remove it.

- [ ] This note is just a placeholder with instructions for you.  Kindly remove it as it is not intended for your users:

![image](https://user-images.githubusercontent.com/57421823/207945961-eea5051d-5bcf-4c73-a8ac-06f39fe41199.png)

- [ ]  On the "License" section of your readme, the link to the license is broken because it points to a file that does not exist in your project.

### Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

## Other topics

- [ ] Your app has no tests.  Kindly add some tests to ensure your app's integrity over time.

- [ ]  The "test" folder still exists, which is used by another test framework that is not Rspec. Kindly consider deleting that folder if you are not using it. 👍

- [ ] Your API is divided among "v1, v2, v3...".  This is your API's version.  Kindly consider using just one version.  In this case, since this is the first version, that would be "v1".